.travis.yml

```diff
@@ -27,11 +27,20 @@ matrix:
 
       # install database
       addons:
-        postgresql: "9.5"
+        postgresql: "12"
+        apt:
+          packages:
+          - postgresql-12
+          - postgresql-client-12
+      env:
```

Comment:
I just added a ClickUp task to move to this style of environment variables: https://app.clickup.com/t/b4m32r

Reply:
I'm not sure but I didn't approach it like that! The way I understood it, PGPORT in the travis.yaml was necessary to travis which port to expose PostgreSQL 12 on, because 5432 is reserved by the default postgres 9 that it comes with. And then PG_PORT in the travis UI was to match what we would put in the .env file, to tell db.js what port to look for the postgres server on